### PR TITLE
Add simple stock poller example

### PR DESCRIPTION
**Pull request by moralesl**
This is an example for a simple stock polling application, that retrieves data from [Twelve Data API](https://twelvedata.com/), stores the latest stock price in DynamoDB and publishes the update to a SQS queue.

This is based on the issue https://github.com/winglang/wing/issues/3563.

### CODE REVIEW

File: examples/stock-poller/Readme.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+## Usage
+
+As the Scheduler component is only available with the `sim` and `tf-aws` provider, you do have to compile and then deploy the application.
+For more details see also [Wing compatability matrix](https://www.winglang.io/docs/standard-library/compatibility-matrix).
+
+### Wing Console
+
+```
+wing it
+```
```
Comment (Chriscbr) [inline examples/stock-poller/Readme.md:24]: To test the application in the Wing Console (using the simulator target), I had to set up a secret at ~/.wing/secrets.json on my computer. We've got some docs about it here: https://www.winglang.io/docs/standard-library/cloud/secret#simulator-sim. Maybe we can add some instructions or link to it here?

File: examples/stock-poller/Readme.md
Comment (Chriscbr) [inline examples/stock-poller/Readme.md:1]: Extra idea (optional): In some examples we've found it helpful to suggest ideas for how people can extend the example. For example "To extend this example, try making the polling application track five ticket symbols, or try displaying the information using the cloud.Website class".

**Review by Chriscbr — APPROVED** [review]
Really cool example @moralesl! I tried it out on my machine using the simulator and it worked out of the box for me. The code is also straightforward to follow. Thanks for including screenshots as well 👍

File: examples/stock-poller/Readme.md (hunk 1 above)
Comment (moralesl) [reply examples/stock-poller/Readme.md:24]: Added below comment to make it more explicit

File: examples/stock-poller/Readme.md
Comment (moralesl) [reply examples/stock-poller/Readme.md:1]: I like the idea very much, I have adopted your comment word for word 😅

**Comment by moralesl** [comment]
Thanks @Chriscbr for your additions, added them to the PR. If everything looks good to you, feel free to merge it 😊

**Pull request merged**